pep-0544.txt

```diff
@@ -1026,9 +1042,8 @@ be considered "non-protocol". Therefore, it was decided to not introduce
 "non-protocol" methods.
 
 There is only one downside to this: it will require some boilerplate for
-implicit subtypes of ``Mapping`` and few other "large" protocols. But, this
-applies to few "built-in" protocols (like ``Mapping`` and ``Sequence``) and
-people are already subclassing them. Also, such style is discouraged for
+implicit subtypes "large" protocols. But, this doesn't apply to "built-in"
+protocols that are all "small". Also, such style is discouraged for
```

Comment:
This appears to define "small" as "has no concrete methods"?

Reply:
I will add a clarification.